The current code causes a compile failure on OSX due to the identifier
CoeffReadCost not being known. It's not clear to me what the correct fix is,
but it's easy to just kill the code and say that we won't unroll our loops
here. No biggie.

--- src/eigen_binary_visitor_plugin.h.orig	2017-02-22 20:28:03.463145979 -0500
+++ src/eigen_binary_visitor_plugin.h	2017-02-22 20:28:17.181254139 -0500
@@ -86,13 +86,7 @@
 			other.rows() == this->rows() && other.cols() == this->cols()
 					&& "DenseBase::visitWith(): inconsistent size.");
 	enum {
-		unroll = SizeAtCompileTime != Dynamic && CoeffReadCost != Dynamic
-				&& (SizeAtCompileTime == 1
-						|| internal::functor_traits < Visitor > ::Cost
-								!= Dynamic)
-				&& SizeAtCompileTime * CoeffReadCost
-						+ (SizeAtCompileTime - 1) * internal::functor_traits
-						< Visitor > ::Cost <= EIGEN_UNROLLING_LIMIT
+		unroll = 0
 	};
 	return BinaryVisitorImpl<Visitor, Derived, OtherDerived,
 			unroll ? int(SizeAtCompileTime) : Dynamic>::Run(derived(), other.derived(),
